src/js/disability-benefits/actions/index.js

```diff
@@ -46,6 +46,22 @@ export function getClaims() {
   };
 }
 
+export function getFilteredClaims(open = true) {
+  return (dispatch) => {
+    makeAuthRequest('/v0/disability_claims',
```

Comment:
Do we need to get new data from the server whenever they change tabs?

If we do, we should probably show a spinner underneath the tabs when while we're waiting.